py/dml/codegen.py

```diff
@@ -3048,12 +3048,19 @@ def stmt_select(stmt, location, scope):
                     else_dead = True
                     break
 
+            iterations = len(clauses)
             if else_dead:
                 (last_cond, last_stmt) = clauses.pop(-1)
                 assert last_cond.constant and last_cond.value
                 if_chain = last_stmt
             else:
                 if_chain = codegen_statement(else_ast, location, scope)
+            if iterations > 50 and isinstance(lst, List):
```

Comment:
I would bump the limit to at least 65 because of "bits in an integer" superstition.

Reply:
Sure. The number's arbitrary anyway.